packages/indexer-client/src/types/clientTypes.ts

```diff
@@ -622,6 +661,9 @@ export interface GetIndexerBlitzPointsResponse {
   tradingPoints: BigDecimal;
   // Total accrued points from referrals for phase 1 & phase 2
   referralPoints: BigDecimal;
+  takerVolumes: BigDecimal;
```

Comment:
What unit is this in? Would be good to specify in a comment.

Additionally, this shouldn't be plural, should it?

Reply:
Ah it must be quote, so on FE we need to adjust. It doesn't have to be. Plural just matches the response.

Reply:
We should prob rename the response to be non-plural to make more sense (can potentially ask backend to tweak as well). It would then be consistent with the likes of `IndexerSubaccountRewardsForProduct`